### PR TITLE
feat-687-location-picker-component

### PR DESCRIPTION
**Pull request by SejoB**
#687

### CODE REVIEW

**Review by daniel-ilan — APPROVED** [review]

**Review by daniel-ilan — APPROVED** [review]

File: src/components/molecules/LocationSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { FC } from 'react';
+import { MapContainer, useMapEvents } from 'react-leaflet';
+import { IPoint } from '../../models/Point';
+import { INITIAL_CENTER } from '../../utils/utils';
+import { Marker } from '../atoms';
```
Comment (yuvalbl) [inline src/components/molecules/LocationSelect.tsx:5]: use absulut imports

File: src/components/molecules/LocationSelect.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  return null;
+};
+
+interface IProps {
+  location: IPoint | null;
+  onLocationChange: (location: IPoint | null) => void;
+}
+//TODO create Map component
+const LocationSelect:FC<IProps> = ({location, onLocationChange}) => {
+
```
Comment (yuvalbl) [inline src/components/molecules/LocationSelect.tsx:30]: remove space

**Review by daniel-ilan — APPROVED** [review]

File: src/components/molecules/SideBarMap.tsx [hunk 3]
```diff
@@ -1,54 +1,21 @@
...
-    return null;
-  }
-  const bounds = getBounds(items);
 
   return (
-    <MapContainer bounds={bounds} zoom={INITIAL_ZOOM} className={classes.wrapper}>
-      <MapViewControl bounds={bounds} center={INITIAL_CENTER} zoom={INITIAL_ZOOM} />
-      <GoogleMapsLayer />
+    <Map data={items}>
       <Marker markerdata={items[0]} />
```
Comment (yuvalbl) [inline src/components/molecules/SideBarMap.tsx:16]: add safe check

File: src/components/molecules/map/Map.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
+import { FC } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+
+import { MapContainer } from 'react-leaflet';
+import L, { LatLng } from 'leaflet';
+import { IPoint } from '../../../models/Point';
+import { INITIAL_CENTER, INITIAL_ZOOM } from '../../../utils/utils';
+import { uniquePoints } from '../../../utils/utils';
+import MapViewControl from '../../../services/MapViewControl';
+import GoogleMapsLayer from './GoogleMapsLayer';
```
Comment (yuvalbl) [inline src/components/molecules/map/Map.tsx:10]: absolute imports

**Review by yuvalbl — APPROVED** [review]

**Pull request merged**